### PR TITLE
To Do Application using React JS

### PR DESCRIPTION
**Pull request by Bishal-35**
# Pull Request for WebMasterLog 💡

## Issue Title: To Do Application using React JS

- **Name of the project** : To Do Application
- **Category:** ReactJs
- **Name:** Bishal Karmakar
- **GitHub ID:** https://github.com/Bishal-35
- **Idenitfy yourself:** Developer

Closes: #10 

### Describe the add-ons or changes you've made 📃

Created a React application using ReactJs which lets the user to create a ToDo List

## Type of change ☑️

What sort of change have you made:

- [X] New feature (non-breaking change which adds functionality)

## How Has This Been Tested? ⚙️

It has been tested it with all the possible ways available.

## Checklist: ☑️

- [X] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [X] This PR does not contain plagiarized content.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly wherever it was hard to understand.
- [X] I have made corresponding changes to the documentation.
- [X] My changes generate no new warnings.
- [X] I have added things that prove my fix is effective or that my feature works.
- [X] Any dependent changes have been merged and published in downstream modules.

## Screenshots 📷

Screenshot 2024-02-20 212451

### CODE REVIEW

File: React-JS-Projects/Basic/ToDo-List/README.md
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Basic/ToDo-List/README.md:1]: @Bishal-35 follow the newly readme template and properly write how to use or run this project.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: React-JS-Projects/Basic/ToDo-List/README.md
Comment (Bishal-35) [reply React-JS-Projects/Basic/ToDo-List/README.md:1]: can you please provide the link for new readme file

Comment (Avdhesh-Varshney) [reply React-JS-Projects/Basic/ToDo-List/README.md:1]: https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/.github/PROJECT-README-TEMPLATE.md

Comment (Avdhesh-Varshney) [reply React-JS-Projects/Basic/ToDo-List/README.md:1]: Code style of readme is still not changed and you have still not written the instructions for how to run this project properly.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Review by Avdhesh-Varshney — APPROVED** [review]
PR Approved 🎉

**Comment by Bishal-35** [comment]
Why is it assigned again?

**Comment by Avdhesh-Varshney** [comment]
@Bishal-35 don't reply directly from email ok. And assigned is required so that it will update on JWOC leaderboard ok.

**Pull request merged**